larpandoracontent/LArTwoDReco/LArCosmicRay/TrackRefinementBaseAlgorithm.cc

```diff
@@ -226,9 +137,8 @@ bool TrackInEMShowerAlgorithm::GetClusterMergingCoordinates(const TwoDSlidingFit
             // ATTN: Set merge point and direction as that at the first layer in the stable region
             if (goodLayerCount == 0)
             {
-                // ATTN: Cluster direction vectors must point to one another
-                clusterMergeDirection = clusterAverageDirection * (isUpstream ? 1.f : -1.f);
-                PANDORA_THROW_RESULT_IF(STATUS_CODE_SUCCESS, !=, clusterMicroFitResult.GetGlobalFitPosition(microIter->second.GetL(), clusterMergePosition));
+                clusterMergeDirection = clusterAverageDirection;
+                clusterMicroFitResult.GetGlobalFitPosition(microIter->second.GetL(), clusterMergePosition);
```

Comment:
What's changed in the decision to no longer throw an exception on an unsuccessful return?

Reply:
this is a mistake, thanks for spotting it!